api-tests/features/inherited-identity.feature

```diff
@@ -18,20 +18,110 @@ Feature: Inherited Identity
       | medium-confidence-pcl250        | kenneth-vot-pcl250-passport       | PCL250            |
       | medium-confidence-pcl250        | kenneth-vot-pcl250-driving-permit | PCL250            |
 
-  Scenario: Migrate PCL 200 HMRC profile successfully with no evidence and returns with P2
-    Given I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity 'alice-vot-pcl200-no-evidence'
+  Scenario Outline: Migrate <expected-identity> HMRC profile successfully with no evidence and returns with <vtr> - requires identity to be proved for stronger profile
```

Comment:
Not super important but this reads a bit weirdly to me... What do you think of something like:
`Migrating a <expected-identity> HMRC profile and returning with <vtr> VTR requires a P2 identity to be proved`

e.g. `Migrating a PCL200 HMRC profile and returning with P2 VTR requires a P2 identity to be proved`

Reply:
reads way better! Thanks :) (why are words so hard)